apis/Google.Cloud.Vision.V1/Google.Cloud.Vision.V1/Google.Cloud.Vision.V1.csproj

```diff
@@ -25,8 +25,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="1.0.0" PrivateAssets="All" />
-    <PackageReference Include="Google.Api.Gax.Grpc" Version="2.1.0" />
-    <PackageReference Include="Grpc.Core" Version="1.6.1" PrivateAssets="None" />
+    <PackageReference Include="Google.Api.Gax.Grpc" Version="2.2.1" />
+    <PackageReference Include="Grpc.Core" Version="1.7.1" PrivateAssets="None" />
```

Comment:
Are you avoiding 1.8 due to the memory-leak issue?

Reply:
Yup - see the commit comment :)

Reply:
Ah yes, I should read more ;)